src/components/Account/Create.tsx

```diff
@@ -33,13 +33,11 @@ import { ucfirst } from '~constants/strings'
 import { Check, Close } from '~theme/icons'
```

Comment:
@selankon `BasicAccountCreation` defined later in this file is not used anymore. Is it still necessary?

Reply:
Well, I supose not. But this is code from old scaffold, take a look here... https://github.com/vocdoni/ui-scaffold/blob/develop/src/components/Account/Create.tsx#L127 should I delete it?